modules/cnv_control_vm_web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-control-vm-web]]
+== starting, stopping, and restarting a virtual machine
```

Comment:
I wanted to put the specific actions in the title so that it was immediately obvious to anyone scanning the contents page. This guide has grown quite large now and I was worried that it would be buried unless made explicit.